.github/dependabot.yml

```diff
@@ -16,10 +16,38 @@ updates:
       - "release-note-none"
     open-pull-requests-limit: 5
 
-  - package-ecosystem: "npm"
-    directory: "/site"
```

Comment:
why do you want to remove this directory?

Reply:
I will revert these changes.